lib/ansible/modules/file.py

```diff
@@ -213,6 +213,14 @@
     path: /etc/foo
     state: absent
 
+- name: Create symlink with custom ownership
+  ansible.builtin.file:
+    state: link
+  become: yes
+  become_user: foo
+  become_method: sudo
+  become_flags: '-g foo'
+
```

Comment:
This is not a functional example and I don't think it does what you says it does:
```"msg": "missing required arguments: path"```